### PR TITLE
scraper for us_ky_mason

### PR DESCRIPTION
**Pull request by sliuu**

### CODE REVIEW

File: recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        return 'MASON_COUNTY_KY'
+
+    def get_enum_overrides(self) -> EnumOverrides:
+        overrides_builder = super(UsKyMasonScraper,
+                                  self).get_enum_overrides().to_builder()
+        overrides_builder.ignore('V', ChargeClass)
+        # Below possibly means (T)hird degree felony, but not sure
+        overrides_builder.ignore('T')
+        overrides_builder.add('10 CASH', BondType.PARTIAL_CASH)
+        overrides_builder.add('WAIVED EXTRADITION',
```
Comment (jamwalla) [inline recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: @ohinds what should we do with this?

Comment (ohinds) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: This is a hold, so we'll need to turn these into a hold on the booking in `populate_data()`.

File: recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        A JailTracker landing page URL ends with: "/jailtracker/index/<INDEX>".
+        This value can either be text or a number. In either case, this method
+        should return the value as a string.
+        """
+        return 'MASON_COUNTY_KY'
+
+    def get_enum_overrides(self) -> EnumOverrides:
+        overrides_builder = super(UsKyMasonScraper,
+                                  self).get_enum_overrides().to_builder()
+        overrides_builder.ignore('V', ChargeClass)
```
[the remark this answers is not in the document]
Comment (ohinds) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: We've been inspecting the context for 'V'. Sometimes it's a probation or parole violation and sometimes it's a driving violation or some other minor offense, which we would label as an INFRACTION.  Looking at the charge name will often disambiguate.

[the remark this answers is not in the document]
Comment (sliuu) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: Updated with the TODO!

File: recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py (hunk 1 above)
Comment (sliuu) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: Sorry, what does this mean? Is there anything I need to change here?

Comment (jamwalla) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: Yep - can you add a `populate_data` method to this scraper? It should call the method from `super()` (i.e. JailTracker's populate_data method) and loop through the bookings. If a booking has a charge with this status (WAIVED EXTRADITION), just call `booking.create_hold()` and delete the charge. Then we'l see that the booking this charge status came from has a hold associated with it. (here's our Hold documentation: https://github.com/Recidiviz/recidiviz-data/tree/master/recidiviz/ingest/models#hold)

Comment (sliuu) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: I added a populate data method and removed the WAIVED EXTRADITION from the enum overrides. Let me know if this was the right thing to do.

File: recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jamwalla) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: Thanks for the TODO - it looks like we should map `V` to `INFRACTION` here, it looks like they're all driving violations. Can you make that one last change?

File: recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py (hunk 1 above)
Comment (jamwalla) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: Yes, that's great!

File: recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sliuu) [reply recidiviz/ingest/scrape/regions/us_ky_mason/us_ky_mason_scraper.py]: Done!

**Comment by jamwalla** [comment]
awesome work! I'll merge this in now.

**Pull request merged**